src/native/eventpipe/ds-sources.c

```diff
@@ -16,6 +16,52 @@
 #include "ds-protocol.c"
 #endif
 
+#undef PORTABLE_RID_ARCH
+
+#if defined(TARGET_X86)
+#define PORTABLE_RID_ARCH "x86"
+#elif defined(TARGET_AMD64)
+#define PORTABLE_RID_ARCH "x64";
+#elif defined(TARGET_ARMV6)
+#define PORTABLE_RID_ARCH "arm32";
+#elif defined(TARGET_ARM)
+#define PORTABLE_RID_ARCH "arm32";
+#elif defined(TARGET_ARM64)
+#define PORTABLE_RID_ARCH "arm64";
+#elif defined(TARGET_S390X)
+#define PORTABLE_RID_ARCH "s390x";
+#elif defined(TARGET_LOONGARCH64)
+#define PORTABLE_RID_ARCH "loongarch64";
+#elif defined(TARGET_POWERPC64)
+#define PORTABLE_RID_ARCH "ppc64le";
+#endif
+
+#undef PORTABLE_RID_OS
+
+#if defined(TARGET_UNIX)
+
+#if defined(TARGET_ANDROID)
+#define PORTABLE_RID_OS "linux-bionic"
+#elif defined(TARGET_LINUX_MUSL)
+#define PORTABLE_RID_OS "linux-musl"
+#elif defined(TARGET_LINUX)
+#define PORTABLE_RID_OS "linux"
+#elif defined(TARGET_OSX)
+#define PORTABLE_RID_OS "osx"
+#else
+#define PORTABLE_RID_OS "unix"
+#endif
+
+#elif defined(TARGET_WINDOWS)
+#define PORTABLE_RID_OS "win"
+#endif
+
+#if defined(PORTABLE_RID_OS) && defined(PORTABLE_RID_ARCH)
+const ep_char8_t* _ds_portable_rid_info = PORTABLE_RID_OS "-" PORTABLE_RID_ARCH;
```

Comment:
I would like to ask if this is actually composing a "portable RID" if it includes the architecture portion. If not, then there's two ways to resolve this:
- Rename it to what it actually represents (non-distro specific RID) but with a better concise descriptive name
- Remove the architecture portion and only report the OS part, namely `linux-bionic`, `linux-musl`, `linux`, `osx`, `win`, and `unix`. The architecture can already be retrieved from the architecture field on the process info structures.

Reply:
Host policy seems to indicate that including architecture allowed for portable RIDs:
- https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostpolicy/deps_format.cpp#L268
- https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostpolicy/deps_format.cpp#L190